src/isar/mission_planner/echo_planner.py

```diff
@@ -75,14 +75,12 @@ def get_mission(self, mission_id: int) -> Mission:
                     f"Failed to create step with exception message: '{str(e)}'"
                 )
                 continue
+            task: Task = Task(steps=[drive_step, *inspection_steps])
+            mission.tasks.append(task)
 
-            mission.steps.append(drive_step)
-            mission.steps.extend(inspection_steps)
-
-        if not mission.steps:
+        if not mission.tasks:
```

Comment:
Check length of `tasks` here and if > 0, instantiate mission

Reply:
`if not list` is the pythonic way of checking if the list is empty.

But the instantiation of the mission may be moved down.

Reply:
Sure, that is better

Reply:
Done